plugins/woocommerce-admin/client/inbox-panel/index.js

```diff
@@ -142,59 +178,78 @@ const renderNotes = ( {
 					);
 				} ) }
 			</TransitionGroup>
+			{ allNotesFetched ? null : notesHaveResolved ? (
```

Comment:
Nested ternary expressions is not like by lint.
Maybe splitting this into two separate conditions would be the best?

Or could we show the `InboxNotePlaceholder` when `allNotesFetched` is `false`?

Reply:
Hmm, not sure how I feel about that particular rule, but I've remedied this in https://github.com/woocommerce/woocommerce/pull/35003/commits/951650873c3334619a4b4c7cb16b64bf58727fc1.